src/manifestoo/commands/list_depends.py

```diff
@@ -11,7 +12,10 @@ def list_depends_command(
     addons_set: AddonsSet,
     transitive: bool = False,
     include_selected: bool = False,
+    addon_sorter: Optional[AddonSorter] = None,
 ) -> Tuple[Iterable[str], Iterable[str]]:
+    if not addon_sorter:
+        addon_sorter = AddonSorterAlphabetical()
```

Comment:
Since the default value of `--sort` is alphabetical we always have a sorter. So let's make this parameter required.

Reply:
Or we can keep it as you propose, to avoid unnecessary churn in tests.